core/src/com/unciv/logic/automation/Automation.kt

```diff
@@ -89,21 +130,23 @@ object Automation {
         if (currentChoice is BaseUnit && !currentChoice.isCivilian()) return city.cityConstructions.currentConstructionFromQueue
 
         var militaryUnits = city.getRuleset().units.values.asSequence()
-            .filter { !it.isCivilian() }
-            .filter { allowSpendingResource(city.civInfo, it) }
+                .filter { !it.isCivilian() }
+                .filter { allowSpendingResource(city.civInfo, it) }
 
         val findWaterConnectedCitiesAndEnemies =
-            BFS(city.getCenterTile()) { it.isWater || it.isCityCenter() }
+                BFS(city.getCenterTile()) { it.isWater || it.isCityCenter() }
         findWaterConnectedCitiesAndEnemies.stepToEnd()
         if (findWaterConnectedCitiesAndEnemies.getReachedTiles().none {
-                (it.isCityCenter() && it.getOwner() != city.civInfo)
-                        || (it.militaryUnit != null && it.militaryUnit!!.civInfo != city.civInfo)
-            }) // there is absolutely no reason for you to make water units on this body of water.
+                    (it.isCityCenter() && it.getOwner() != city.civInfo)
+                            || (it.militaryUnit != null && it.militaryUnit!!.civInfo != city.civInfo)
+                }) // there is absolutely no reason for you to make water units on this body of water.
```

Comment:
Are all these extra indentations android studio auto-formatting? I personally don't like them tbh, and the kotlin style guide agrees with me on that. Its a style thing so this won't affect my approval regardless

Reply:
Yeah, did an Android Studio auto format pass. Can clean up a bit

Reply:
I even think the defaults for that changed ArcticFox - Bumblebee - Chipmunk. But - we could make these settings part of the project under version control.

Reply:
Or at least mention in the Wiki. Done on my editor and fixed